COMPILATION.md

```diff
@@ -15,7 +15,10 @@ Keep in mind using the pre-built packages when available.
 * libcurl
 * libxml2
 * openssl
-* /etc/mime.types (the package providing depends on the OS)
+* mime.types (the package providing depends on the OS)
+	* /etc/mime.types is default for not macOS
+	* /etc/apache2/mime.types is default for macOS
```

Comment:
This doesn't seem to match what you say on the PR description:
> On non-macOS, the default mime.types path is /etc/mime.types. (this path is used when this option is not specified)
> On macOS after Cataline, apache2.4 is pre-installed , so /etc/apache2/mime.types is the default for macOS.

So if this is true, I suggest:

> 	* /etc/mime.types is default GNU/Linux, BSD and macOS until Catalina (10.15)
> 	* /etc/apache2/mime.types is default for macOS Catalina (10.15) and newer

Reply:
I'll fix it because your description is better.
Thanks a lot.

Reply:
Same as above.